pkgs/development/libraries/libnghttp2_asio/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, cmake
+, nghttp2, openssl, boost
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libnghttp2_asio";
+  version = "unstable-2022-08-11";
+
+  src = fetchFromGitHub {
+    owner = "nghttp2";
+    repo = "nghttp2-asio";
+    rev = "e877868abe06a83ed0a6ac6e245c07f6f20866b5";
+    sha256 = "sha256-XQXRHLz0kvaIQq1nbqkJnETHR51FXMB1P9F/hQeZh6A=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    boost
+    nghttp2
+    openssl
+  ];
+
+  outputs = [ "out" "dev" "doc" ];
```

Comment:
we normally write output near the top between version and src

Reply:
Fixed.